### PR TITLE
Iteration 1

### PR DESCRIPTION
**Pull request by nkumar290**
## Story
https://techstars-engineering.atlassian.net/jira/software/c/projects/KERNEL/boards/14?modal=detail&selectedIssue=KERNEL-174

## Background

- use new ref data pattern 
- create table structure
- leads csv: [Master Pipeline Leads](https://docs.google.com/spreadsheets/d/1g9n-aBmLMu-hFMPg-t5H_WAde21Zm90su6WTo2CqSyw/edit#gid=0) 
- founder catalyst applicants csv: research.FC_applicants
- founder catalyst attendees csv: research.FC_participants
- add argument for kill/fill vs append
- whitelist tables that this can write to

## Testing
A description of what testing was done to verify it was working
- unit testing
- manual testing:
    - locally
    - dev environment

### CODE REVIEW

File: .python-version [hunk 1]
```diff
@@ -1 +0,0 @@
-3.9.9
```
Comment (peytontrask) [inline .python-version:1]: We need to test this with 3.10 I'm pretty sure everything else has been upgraded

File: src/main.py [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+        if (
+            file.endswith('.csv')
+            and ((table and file.lower() == table.lower() + '.csv')
+                 or (not table))
+        ):
+            logger.info(f'working on {file}')
+            # import csv file
+            df_csv = pd.read_csv(f'{data_folder}//{file}')
+            table_name = file.replace(".csv", "")
+            # WHITELIST of schemas to write to
```
Comment (peytontrask) [inline src/main.py:36]:
```suggestion
            # WHITELIST of schemas to write to 
            # TODO: whitelist schemas (not tables) and parse from arguments with a default of research
```

File: src/main.py [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+
+if __name__ == "__main__":
+    # initial setup -- gets current state of ref tables and creates a CSV.  Replaces what's in data/reference folder
+    # table_to_csv('industry_vertical_evo', engine, schema, data_folder)
+    # define the data folder
+    data_folder = os.getenv("DATA_FOLDER")
+
+    # parse args
+    if len(sys.argv) > 3 or len(sys.argv) < 2:
```
Comment (peytontrask) [inline src/main.py:50]: https://docs.python.org/3/library/argparse.html

This library may be more robust for executing cli args with scripts

File: .python-version (hunk 1 above)
Comment (nkumar290) [reply .python-version:1]: python-version file is being deleted. The tsdata-python-microservice repo was used as the template for this repo however we need to change the repo structure (files and folders) and code pattern to match tsdata-reference. there is no .python-version file in the  tsdata-reference repo, i have already tested with 3.10

File: src/main.py (hunk 3 above)
Comment (nkumar290) [reply src/main.py:51]: can we implement the argsparser lib in the next iteration? would like to get this over the finish line before end of sprint

Comment (nkumar290) [reply src/main.py:51]: I can create a new 1 point story to remove the table white list and implement argsparser

Comment (peytontrask) [reply src/main.py:51]: yes for sure

**Review by peytontrask — APPROVED** [review]

**Pull request merged**